### PR TITLE
DS 889: Drupal Admin UI Paternlab Section

### PR DESCRIPTION
**Pull request by MarcinMr**
## Jira

https://pegadigitalit.atlassian.net/browse/DS-889

## Summary

There's a new section Drupal Admin UI in the pattern lab navigation which contains the UX builds of the Drupal Admin panels for the new Layout Builder project.

## Details

**1. In the Bolt, we have a new section with UX builds:**
 - text paragraph
 - advanced text paragraph
 - icon paragraph
 - image paragraph
 - button paragraph
 - blockquote paragraph
 - list paragraph
 - action blocks paragraph
 - tabs and accordion paragraph
 
 These above were recreated from the private Codepen.
 
 **2. The animation tab was updated** 
 - the animation `type` dropdown was removed
 - the animation `style` was updated to have only the `fade` and `bounce` for now
 
:point_right: _**it's possible that this will be updated again in the future**_

## How to test

Pull the branch. Look at the Drupal admin UI section. Go thru all component examples and confirm that there are the same as Codepen examples.

## Release notes

We have a new Drupal admin UI section in the pattern lab navigation.

### CODE REVIEW

**Comment by MarcinMr** [comment]
@remydenton 

I transferred Codepen's examples to the Bolt, it's ready to take a look at this

**Review by remydenton — CHANGES_REQUESTED** [review]
This looks really nice @MarcinMr , nice work.  I have the following updates:
1. One nitpitpick inline comment about a typo
2. Updates to make to the animations tab that I posted on the JIRA ticket so Matt could see: https://pegadigitalit.atlassian.net/browse/DS-889?focusedCommentId=65673 
3. One additional question, should we make this new section hidden/shown alongside the `test` and `archive` sections?  My thinking is that it is really only for a few people (me, Andrew, Matt), whereas the rest of the system should be useful to most developers and designers.  Thoughts on that @colbytcook @danielamorse ?

File: docs-site/src/pages/pattern-lab/_patterns/70-drupal-admin-user-interface/35-tabs-and-accrodion-paragraph.twig [hunk 1]
```diff
@@ -0,0 +1,121 @@
+{% extends "_drupal-admin-ui-template.twig" %}
```
[the remark this answers is not in the document]
Comment (MarcinMr) [reply docs-site/src/pages/pattern-lab/_patterns/70-drupal-admin-user-interface/35-tabs-and-accrodion-paragraph.twig:1]: Regards point 3, I was also wondering if this should be hidden or visible

[the remark this answers is not in the document]
Comment (remydenton) [reply docs-site/src/pages/pattern-lab/_patterns/70-drupal-admin-user-interface/35-tabs-and-accrodion-paragraph.twig:1]: Let's discuss briefly in dev huddle

**Review by remydenton — APPROVED** [review]
Updates look good, thank you @MarcinMr.

For what it's worth, I had the same behavior in your screencast (wrong tabs display) when not in incognito mode.  After I cleared local storage in developer tools, it worked as expected.

**Pull request merged**